NeuroID/src/advancedDeviceAndroidLib/java/com/neuroid/tracker/service/NIDEventSender.kt

```diff
@@ -0,0 +1,54 @@
+package com.neuroid.tracker.service
```

Comment:
It seems like we have this across all out flavors, can we not put this in the main lib?

Reply:
Unfortunately the ReactNative and AndroidLib implementations are slightly different. The ReactNative lib has deprecated requestBody() and added an extension to String that creates the requestBody from there. I think the Facebook reactnative lib did this.

Reply:
Unfortunately the ReactNative and AndroidLib implementations are slightly different. The ReactNative lib has deprecated requestBody() and added an extension to String that creates the requestBody from there. I think the Facebook reactnative lib did this.

Reply:
was able to move to main, removed from flavors. explicitly added okhttp client version so both native and ReactNative use the same OKHTTP version.